components/frontend/src/components/ui/ask-user-question.tsx

```
@@ -0,0 +1,345 @@
+                  return (
+                    <button
+                      key={idx}
+                      onClick={() => setActiveTab(idx)}
+                      className={cn(
+                        "flex items-center gap-1 px-2 py-0.5 rounded text-xs font-medium transition-colors",
+                        active
+                          ? "bg-foreground/10 text-foreground"
+                          : "text-muted-foreground hover:text-foreground hover:bg-foreground/5",
+                        done && !active && "text-green-600 dark:text-green-400"
+                      )}
+                    >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)